components/header.tsx

```diff
@@ -6,17 +6,19 @@ import Tooltip from './tool-tip';
 import { onEntryChange } from '../contentstack-sdk';
 import { getHeaderRes } from '../helper';
 import Skeleton from 'react-loading-skeleton';
+import { HeaderProps, Entry, Links } from "../typescript/layout";
+
+export default function Header({ header, entries }: {header: HeaderProps, entries: Entry}) {
 
-export default function Header({ header, entries }) {
   const router = useRouter();
   const [getHeader, setHeader] = useState(header);
 
-  function buildNavigation(ent, hd) {
+  function buildNavigation(ent: Entry, hd: any) {
```

Comment:
header response should not be any & can be provided as an object(refer to type created for header)